mandatory/4-tax.js

```diff
@@ -17,7 +21,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let total = calculateSalesTax(price).toFixed(2);
+  return "£" + total;
+}
 
```

Comment:
Very clear and concise code in this one. Well done.

Reply:
Thank you Melese :)

Reply:
Thanks Christina :)